packages/story-editor/src/components/checklist/checks/publisherLogoExistence.js

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import { useCallback } from '@web-stories-wp/react';
+import { List, THEME_CONSTANTS } from '@web-stories-wp/design-system';
+
+/**
+ * Internal dependencies
+ */
+import { useStory } from '../../../app';
+import { states, useHighlights } from '../../../app/highlights';
+import { ChecklistCard, ChecklistCardStyles } from '../../checklistCard';
+import { PRIORITY_COPY } from '../constants';
+import { useRegisterCheck } from '../countContext';
+import { useIsChecklistMounted } from '../popupMountedContext';
+
+export function publisherLogoExistence(publisherLogo) {
+  return !publisherLogo?.url && !publisherLogo?.height && !publisherLogo?.width;
+}
```

Comment:
```suggestion
export function publisherLogoExistence(publisherLogo) {
  return !(publisherLogo?.id);
}
```

Reply:
Also, FWIW, this function checks whether the logo is **missing**, but the function is named publisherLogo_Existence_. That is super confusing.

Either rename this to `publisherLogoInexistence` (which is also confusing), or reverse the check:

```suggestion
/**
 * Determines whether a publisher logo exists.
 * @param {Object} publisherLogo Publisher logo object.
 * @return {boolean} Whether a publisher logo exists.
 */
export function publisherLogoExistence(publisherLogo) {
  return Boolean(publisherLogo?.id);
}
```

Of course update `PublisherLogoExistence` below accordingly

Reply:
+1 for `PublisherLogoMissing`

Reply:
Ah good shout